hedera-node/hedera-app/build.gradle.kts

```diff
@@ -44,6 +44,7 @@ dependencies {
     implementation(project(":hedera-node:hedera-util-service-impl"))
 
     implementation(libs.bundles.swirlds)
+    implementation(libs.commons.codec)
```

Comment:
What API from this dependency do we need?

Reply:
Any reason to keep this around as a comment if the dependency isn't needed anymore?

Reply:
Oops, removed it.